### ISSUE TITLE
Add a combo .mangle and .markov!

### ISSUE THREAD

**Issue by Lokosmos** [flashcactus/kountdown#11]
Or possibly look into how to call commands using the .py command, so we can do it ourselves!

**Comment by flashcactus** [comment]
nah, the .py stuff is run through a webservice, so no chance here.
